config/values.yml

```diff
@@ -4,5 +4,25 @@ system_domain: ""
 system_namespace: ""
 
 images:
-  metric_proxy: "oratos/metric-proxy@sha256:76fe51d101ca0488996b7b17b8f3a28388996c694e4688da6156e9c3474b8bbc"
+  metric_proxy: "oratos/metric-proxy@sha256:dd340c789169485f256b36d4552182dfa789f82bb5e5e95b4d709b1decf72734"
```

Comment:
Which image is this? I don't think it's the one from the pipeline?

Reply:
This is the image from before the TLS removal commits. We could switch to the pipeline image.